images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -96,7 +96,8 @@ ARG SDK_STORAGE_ENV_NAME
 ARG SDK_STORAGE_BASE_URL_VALUE
 WORKDIR /
 
-ENV PATH="$PATH:/opt/oryx:/opt/nodejs/lts/bin:/opt/yarn/stable/bin:/opt/hugo"
+ENV ORYX_PATHS="/opt/oryx:/opt/nodejs/lts/bin:/opt/yarn/stable/bin:/opt/hugo/lts"
```

Comment:
Adding an extra environment variable which contains all paths related to Oryx. Came across a scenario from jam stack team where their container's path was being overwritten causing Oryx executables not being found (since path was modified). This new environment variable will allow jam stack team to always make sure (in their entrypoint script) oryx paths are present